integrationtest/kube/collect_kube_test.go

```diff
@@ -527,8 +528,15 @@ dremio-jfr-time-seconds: 10
 	sort.Strings(foundFiles)
 	sort.Strings(expectedFiles)
 
-	if !reflect.DeepEqual(expectedEmptyFiles, foundEmptyFiles) {
-		t.Errorf("Expected the following files to be empty:\n %v\n But found the following:\n %v", expectedEmptyFiles, foundEmptyFiles)
+	for _, expectedEmptyFile := range expectedEmptyFiles {
+		for _, foundEmptyFile := range foundEmptyFiles {
+			if expectedEmptyFile == foundEmptyFile {
```

Comment:
I'm not sure I fully grok what is going on here..doesn't this mean it will just always pass?

Reply:
so this is basically saying we check each empty file we want against the list of found files, if its there we add it into a 3rd list `compareEmptyFiles` - this means at the end we compare our list of expected empty files against the list of files we found to be empty and we should have a match. If not then there would have been one missing.

The reason I'm doing this here is that sometimes container logs like "wait for zookeeper" may be empty, it may not, so we cant directly compare (deep equals) all the empty files we found against the ones we want empty as if a container log we dont care about is empty it will fail the check